src/migration/prod/locals.tf

```diff
@@ -0,0 +1,42 @@
+locals {
+  prefix             = "io"
+  env_short          = "p"
+  location_short     = { westeurope = "weu", italynorth = "itn", germanywestcentral = "gwc", northeurope = "neu" }
+  project_itn        = "${local.prefix}-${local.env_short}-${local.location_short.italynorth}"
+  project_weu        = "${local.prefix}-${local.env_short}-${local.location_short.westeurope}"
+  project_weu_legacy = "${local.prefix}-${local.env_short}"
+  secondary_project  = "${local.prefix}-${local.env_short}-${local.location_short.germanywestcentral}"
+
+  environment = {
+    prefix          = local.prefix
+    env_short       = local.env_short
+    location        = "italynorth"
+    domain          = "eng"
+    app_name        = "migitn"
+    instance_number = "01"
+  }
+
+  tags = {
+    CostCenter  = "TS310 - PAGAMENTI & SERVIZI"
+    CreatedBy   = "Terraform"
+    Environment = "Prod"
+    Owner       = "IO"
+    Source      = "https://github.com/pagopa/io-infra/blob/main/src/migration/prod"
+  }
+
+  storage_accounts = [
```

Comment:
should this list be updated once the storage accounts in itn have been created?

Reply:
Yes, exactly! Or we can decide to instantiate the module close to the storage accounts to be migrated